src/errorMsgs.js

```diff
@@ -43,19 +48,53 @@ const findErrors = (function () {
                 continue;
             }
             const src1 = node.src[1];
-            if( 1 < node.src.length && 0 < src1.value.indexOf('0') ) {
+            if( 1 < node.src.length && src1.value.length == 3 && 0 < src1.value.indexOf('0') ) { // vc0
```

Comment:
would it just be easier to just say:

`src1.value == 'vc0'`?

Is that the only thing you are looking for with :
`src1.value.length == 3 && 0 < src1.value.indexOf('0')`

Reply:
Yes, updated.